BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -278,6 +264,22 @@ class BraintreeClient @VisibleForTesting internal constructor(
         }
     }
 
+    /**
+     * @suppress
+     */
+    @Suppress("TooGenericExceptionCaught")
+    private fun sendGraphQLPOSTSync(
+        json: JSONObject?,
+        configuration: Configuration
+    ): Pair<String?, Exception?> = try {
+        val data = json?.toString()
+        val response = graphQLClient.post("", data, configuration, authorization)
```

Comment:
Nit: we have an overload graphQLClient `post` method that doesn't take in `path` and sends empty string in the method - can we use that `post` method here instead?